trio/_core/_tests/test_run.py

```diff
@@ -777,20 +796,20 @@ async def task2():
 
     # Trying to exit a cancel scope from an unrelated task raises an error
     # without affecting any state
-    async def task3(task_status):
+    async def task3(task_status: _core.TaskStatus[_core.CancelScope]) -> None:
         with _core.CancelScope() as scope:
             task_status.started(scope)
             await sleep_forever()
 
     async with _core.open_nursery() as nursery:
-        scope = await nursery.start(task3)
+        scope: _core.CancelScope = await nursery.start(task3)
```

Comment:
Why the explicit type annotation?

Reply:
Mypy demanded an annotation here, since `nursery.start()` isn't properly typed - it doesn't know what it returns yet.

Reply:
Makes sense.